webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import Select from '../../../../../common/forms/Select';
+import Checkbox from '../../../../../common/forms/Checkbox';
+import NumericInput from '../../../../../common/forms/NumericInput';
+import './disk.scss';
+
+const Disk = ({
+  removeDisk,
+  updateDisk,
+  name,
+  config: { datastores, storagePods, diskModeTypes, vmExists },
+  storagepod,
+  datastore,
+  sizeGb,
+  thin,
+  eagerzero,
+  mode
+}) => {
+  return (
+    <div className="disk-container">
+      <div className="form-group">
+        <label className="col-md-2 control-label">
+          {__('Disk name')}
+        </label>
+        <div className="col-md-4">
+          {name}
+        </div>
+        <div className="col-md-2">
+          <button
+            type="button"
+            className="close"
+            disabled={ vmExists }
+            onClick={removeDisk}
+            aria-label="Close"
+          >
+            <span aria-hidden="true">×</span>
+          </button>
+        </div>
+      </div>
+      {!datastore &&
+        <Select
+          label={__('Storage Pod')}
+          value={storagepod}
+          disabled={vmExists}
+          onChange={updateDisk.bind(this, 'storagepod')}
+          options={storagePods}
+        />}
+      {!storagepod &&
+        <Select
+          disabled={vmExists}
+          label={__('Data store')}
+          value={datastore}
+          onChange={updateDisk.bind(this, 'datastore')}
+          options={datastores}
+        />}
+
+      <Select
+        label={__('Disk Mode')}
+        value={mode}
+        disabled={vmExists}
+        onChange={updateDisk.bind(this, 'mode')}
+        options={diskModeTypes}
+      />
+
+      <NumericInput
+        value={sizeGb}
+        format={v => `${v} GB`}
```

Comment:
The value is displayed like this: "55 gb". Can we make the GB actually in capital letters?